components/GlobalStyles.js

```diff
@@ -0,0 +1,37 @@
+import { createGlobalStyle } from 'styled-components';
+
+const GlobalStyles = createGlobalStyle`
+  * {
+font-family: 'Open Sans' , sans-serif;
+  }
+
+  body {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    height: auto;
+    background-color: whitesmoke;
+    color: black;
```

Comment:
a little note :) You don't need to write this, because the default value is always black :) You can try it out, it should not change anything if you make it out, since you anyway in other places in your style yes the color also assigns again

Reply:
right ;)